src/StoreApi/Routes/V1/Products.php

```diff
@@ -281,6 +281,28 @@ public function get_collection_params() {
 			'validate_callback' => 'rest_validate_request_arg',
 		);
 
+		// If the $_REQUEST contains a taxonomy query, add it to the params and sanitize it.
+		foreach ( $_REQUEST as $param => $value ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			if ( str_starts_with( $param, 'tax_' ) && ! str_ends_with( $param, '_operator' ) ) {
+				$params[ $param ] = array(
+					'description'       => __( 'Limit result set to products assigned a specific category ID.', 'woo-gutenberg-products-block' ),
+					'type'              => 'string',
+					'sanitize_callback' => 'wp_parse_id_list',
+					'validate_callback' => 'rest_validate_request_arg',
+				);
+			}
+			if ( str_starts_with( $param, 'tax_' ) && str_ends_with( $param, '_operator' ) ) {
+				$params[ $param ] = array(
+					'description'       => __( 'Operator to compare product category terms.', 'woo-gutenberg-products-block' ),
+					'type'              => 'string',
+					'enum'              => [ 'in', 'not in', 'and' ],
```

Comment:
I am wondering if this should be `not_in` with an underscore?

Reply:
Good catch! Since the `$operator_mapping` has it as `not_in` I have updated category, tag, attribute, and taxonomy operator enum from `not in` to `not_in` 
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/5be9023bcbc1918f84e9951f0acca114dcca99d4/src/StoreApi/Utilities/ProductQuery.php#L92-L96

[604cb1a](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6152/commits/604cb1a61fa34f7468d11924d9a0b81c7d800e8a)
[38d6edd](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6152/commits/38d6edd312edc24dee0259336c997970ba8fc415)